src/components/clusters/ClusterDetailsMultiRegion/components/MachinePools/components/EditMachinePoolModal/fields/__tests__/WindowsLicenseIncludedField.test.tsx

```diff
@@ -0,0 +1,193 @@
+  WindowsLIDisabledMachinePool,
+  WindowsLIEnabledMachinePool,
+} from './WindowsLicenseIncludedField.fixtures';
+
+const {
+  WINDOWS_LICENSE_INCLUDED_AWS_DOCS: AWS_DOCS_LINK,
+  WINDOWS_LICENSE_INCLUDED_REDHAT_DOCS: REDHAT_DOCS_LINK,
+} = links;
+
+// Formik Wrapper:
+const buildTestComponent = (
+  initialValues: FormikValues,
+  children: React.ReactNode,
+  onSubmit: () => void = jest.fn(),
+  formValues = {},
+) => (
+  <Formik
+    initialValues={{
+      ...initialValues,
+      ...formValues,
+    }}
+    onSubmit={onSubmit}
+  >
+    {children}
+  </Formik>
+);
+
+describe('<WindowsLicenseIncludedField />', () => {
+  describe('When creating a new Machine Pool', () => {
+    describe('When selected Machine Type is Windows LI compatible', () => {
+      it('Shows the checkbox text', async () => {
+        // Arrange
+        render(
+          buildTestComponent(
+            { initialValuesWithWindowsLIEnabledMachineTypeSelected },
+            <WindowsLicenseIncludedField />,
+          ),
+        );
+
+        // Act
+        // Assert
+        expect(
+          screen.getByText('Enable machine pool for Windows License Included'),
+        ).toBeInTheDocument();
+
+        const checkbox = screen.getByRole('checkbox');
+        expect(checkbox).toBeInTheDocument();
+      });
+    });
+
+    it('Shows a PopoverHint for Machine Pools which are Windows LI enabled and verifies its functionality', async () => {
+      // Arrange
+      const { user } = render(
+        buildTestComponent(
+          { initialValuesWithWindowsLIEnabledMachineTypeSelected },
+          <WindowsLicenseIncludedField />,
+        ),
+      );
+
+      // Act
+      const popoverHint = screen.getByLabelText('More information');
+      await user.click(popoverHint);
+
+      // Assert
+      expect(screen.getByText(/Learn more about/i)).toBeInTheDocument();
+
+      const awsDocsLink = screen.getByText('Microsoft licensing on AWS');
+      const redhatDocsLink = screen.getByText('how to work with AWS-Windows-LI hosts');
+      expect(awsDocsLink).toBeInTheDocument();
+      expect(redhatDocsLink).toBeInTheDocument();
+      expect(awsDocsLink).toHaveAttribute('href', AWS_DOCS_LINK);
+      expect(redhatDocsLink).toHaveAttribute('href', REDHAT_DOCS_LINK);
+
+      expect(
+        screen.getByText(
+          'When enabled, the machine pool is AWS License Included for Windows with associated fees.',
+        ),
+      ).toBeInTheDocument();
+    });
+
+    it('Shows a disabled checkbox with a related tooltip for Machine Types which are NOT Windows LI compatible', async () => {
+      // Arrange
+      const { user } = render(
+        buildTestComponent({ initialValues }, <WindowsLicenseIncludedField />),
+      );
+
+      // Act
+      // Assert
+      const checkbox = screen.getByText('Enable machine pool for Windows License Included');
+      expect(checkbox).toBeInTheDocument();
+      expect(checkbox).toHaveClass('pf-m-disabled');
```

Comment:
we shouldn't rely on classnames to test state - isn't there another way of checking for 'disabled' state?